File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    printf("Length: ");
    while (true)
    {
        int length = get_int();
        if (length > 2010 || length < 0)
        {
            printf("Retry: ");
            continue;
        }

        int i;
        for (i = 1; i < length + 1; i++)
        {
            char space = ' ';
            int n = length - i;
            
            int b;
            for (b = 0; b < n; b++)
            {
                printf("%c", space);
            }
            
            int c;
            for (c = 0; c < i; c++)
            {
                printf("#");
            }
            
            int d;
            for (d = 0; d < 2; d++)
            {
                printf("%c", space);
            }
            
            int e;
            for (e = 0; e < i; e++)
            {
                printf("#");
            }
            printf("\n");

        }
    
        break;
    }
}

File: pset6/mario/mario.py
from cs50 import *
# Mario.py
while True:

    length = get_int("Length:")

    if length > 23 or length <= 0:
        continue

    print('\n' * 24)

    for i in range(1, length + 1):
        space = ' '
        n = length - i
        print(space * n, end = '')
        print('#' * i, end = '')
        print(space * 2, end = '')
        print('#' * i)

    break
